onboarding/tests/test_scaffolding.py

```diff
@@ -2,6 +2,21 @@
 
 from onboarding.scaffolding import OnboardingScaffolding, get_scaffolding, update_scaffolding
 from onboarding.schema import session_key_for_step
+from findhelp.tests.factories import LngLats
+
+
+@pytest.fixture
+def fake_is_lnglat_in_nyc(monkeypatch):
+    from onboarding import scaffolding
+
+    def is_lnglat_in_nyc(point):
+        return {
+            LngLats.BROOKLYN_HEIGHTS: True,
+            LngLats.ALBANY: False,
+            LngLats.YONKERS: False,
+        }[point]
+
+    monkeypatch.setattr(scaffolding, "is_lnglat_in_nyc", is_lnglat_in_nyc)
```

Comment:
We're just going to fake this function so we don't have to reload all the borough data into the database.  This makes our tests in this module more like pure unit tests anyways--the actual tests for `is_lnglat_in_nyc` over in findhelp will test that function works as we expect.